monitor/base.go

```diff
@@ -18,8 +18,8 @@ func logHeader(logger log.Logger, header *types.Header) {
 	logger.Debug("RECEIVED_HEADER", "blockNumber", header.Number)
 }
 
-func logTransaction(logger log.Logger) {
-	logger.Debug("RECEIVED_TRANSACTION")
+func logTransaction(logger log.Logger, params ...interface{}) {
+	logger.Info("RECEIVED_TRANSACTION", params...)
```

Comment:
maybe debug was on purpose?

Reply:
I think it made sense to have it as debug as it didn't have any data, now we want that to reach the logs